src/MissionManager/VisualMissionItem.h

```diff
@@ -194,7 +194,7 @@ class VisualMissionItem : public QObject
     void altDifferenceChanged           (double altDifference);
     void altPercentChanged              (double altPercent);
     void terrainPercentChanged          (double terrainPercent);
-    void terrainCollisionChanged        (double terrainCollision);
+    void terrainCollisionChanged        (bool terrainCollision);
```

Comment:
How come this wasn't causing signal connect compiler failures. Is nobody using this no the C++ side? Wondering if there is some connect code somewhere.

Reply:
I tried to find the connect() with the signal and had no luck. It's the reason why it compiles successfully.

Reply:
OK, thanks. That means it's only used on the Qml side.